src/python/pants/backend/python/tasks/python_task.py

```diff
@@ -187,3 +192,12 @@ def _chroot_path(self, interpreter, pex_info, targets, platforms, extra_requirem
 
     fingerprint = hash_utils.hash_all(fingerprint_components)
     return os.path.join(self.chroot_cache_dir, fingerprint)
+
+  def _configure_shebang(self, targets, path, interpreter, pex_info):
+    builder = PEXBuilder(path=path, interpreter=interpreter, pex_info=pex_info, copy=True)
```

Comment:
should probably rename this method to e.g. `_setup_builder` since it's doing much more than just configuring the shebang?